File: Programmers/Level1/로또의최고순위와최저순위/로또의최고순위와최저순위.js
function solution(lottos, win_nums) {
    const lotto_point = [6, 6, 5, 4, 3, 2, 1];
    const zero_count = lottos.filter((v) => v === 0).length;
    const count = lottos.filter((v) => win_nums.includes(v)).length;

    return [lotto_point[count + zero_count], lotto_point[count]];
}

File: Programmers/Level1/로또의최고순위와최저순위/로또의최고순위와최저순위.py
def solution(lottos, win_nums):
    lst = [6, 6, 5, 4, 3, 2, 1]
    count = 0
    for x in win_nums:
        if x in lottos:
            count += 1 

    return [lst[lottos.count(0) + count], lst[count]]
